x-pack/plugins/timelines/public/components/t_grid/body/control_columns/checkbox.tsx

```diff
@@ -18,7 +19,10 @@ export const RowCheckBox = ({
   columnValues,
   disabled,
   loadingEventIds,
+  data,
 }: ActionProps) => {
+  const ruleProducers = data.find((d) => d.field === ALERT_RULE_PRODUCER)?.value ?? [];
+  const ruleProducer = ruleProducers[0];
```

Comment:
This may return undefined, would that be a problem?

Reply:
I thought about that as well. But specifically in our tests no, since we load archive data and all entries have a value for the producer.